MtlMicro Meeting: December 2017
Quince et al., 2017
"DESMAN: a new tool for de novo extraction of strains from metagenome"
========================================================
author: P.Tran
date: December 6, 2017
autosize: true

WELCOME
========================================================
Welcome to the First MtlMicro Meeting!
Thank you all for the paper suggestions.
This paper was suggested by Nicolas Tromas from the Shapiro Lab. 

ABSTRACT
========================================================

Problem:
Large multi-sample metagenomes are generated, but strain variation results in fragmentary co-assemblies
Therefore, the bins (MAGs) are can be an aggregate of closely related strains.

What DESMAN is:
a tools that identifies varients in core genes and uses co-occurence accross samples to link these variants into haplotypes and abundance profiles

How this was tested:
- in silico
- A complex 50-species 210-genome 96-samples synthetic mock data set and applied to the Tara Oceans Microbiome

Overview of the DESMAN pipeline
====================================
![alt text](12_2017_Quince_et_al_2017-figure/Quince_et_al_2017_Figure1.gif)

Figure 1.

Some key terms
====================================
>"We developped a full **Bayesian Model**, fitted by a **Markov Chain Monte Carlo (MCMC)** **Gibbs sampler**, to learn the strain frequencies, their **haplotypes** and allso sequencing error rates. To improve **convergence**, we initialize the Gibbs sampler using a **non-negative matrix factorisation** (NMF), or more properly **non-negative tensor factorisation (NTF)**, a method from machine learning that is equivalent to the **maximum likelihood solution**" 

(Second page of PDF, Quince et al. 2017)

Definitions
====================================
Bayesian Model:
MCMC:
Gibbs Sampler: 
Convergence:
Non-negative matrix factorisation:
Non-negative tensor factorisation:
Maximum-likelihood:



